testcases/crypto/ec_func.c

```diff
@@ -2536,7 +2531,7 @@ struct btc_test {
 };
 
 static const struct btc_test btc_tests[] = {
-    { .ec = { &secp256k1, sizeof(secp256k1), CK_FALSE, CURVE_PRIME,
+    { .ec = { &secp256k1, sizeof(secp256k1), CK_FALSE, CURVE_KOBLITZ,
```

Comment:
Also change the curve type in the next block below (?):
```
 { .ec = { &secp256k1, sizeof(secp256k1), CK_FALSE, CURVE_PRIME,
              CURVE256_LENGTH, "secp256k1" },
  ...
```

Reply:
Oups, yes, of course .... Although it really does not make any difference for BTC tests.